app-modules/service-management/database/migrations/2024_12_30_091625_add_customers_auditors_managers_fields_in_service_request_types_table.php

```diff
@@ -0,0 +1,91 @@
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::table('service_request_types', function (Blueprint $table) {
+            $table->boolean('is_managers_service_request_update_email_enabled')->after('is_managers_service_request_resolved_notification_enabled')->default(false);
+            $table->boolean('is_managers_service_request_update_notification_enabled')->after('is_managers_service_request_update_email_enabled')->default(false);
+            $table->boolean('is_managers_service_request_status_change_email_enabled')->after('is_managers_service_request_update_notification_enabled')->default(false);
+            $table->boolean('is_managers_service_request_status_change_notification_enabled')->after('is_managers_service_request_status_change_email_enabled')->default(false);
+            $table->boolean('is_auditors_service_request_update_email_enabled')->after('is_auditors_service_request_resolved_notification_enabled')->default(false);
+            $table->boolean('is_auditors_service_request_update_notification_enabled')->after('is_auditors_service_request_update_email_enabled')->default(false);
+            $table->boolean('is_auditors_service_request_status_change_email_enabled')->after('is_auditors_service_request_update_notification_enabled')->default(false);
```

Comment:
`after` is not available to us since we use Postgres and can be removed.

Reply:
Done